src/main/java/codesquad/springcafe/controller/UserController.java

```diff
@@ -0,0 +1,47 @@
+package codesquad.springcafe.controller;
+
+import codesquad.springcafe.model.User;
+import codesquad.springcafe.service.UserService;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/users")
+public class UserController {
+
+    private final UserService userService;
+
+    @Autowired
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @PostMapping
+    public String signUp(@ModelAttribute("user") User user, Model model) {
```

Comment:
```suggestion
    public String signUp(@ModelAttribute("user") UserDto user, Model model) {
```

아래 코드가 의미가 있으려면 위쪽 코드도 이렇게 되야 할 거 같아요.